src/providers/ConfigProvider.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { createContext, FunctionComponent, ReactNode, useEffect, useState } from 'react';
```

Comment:
I think this file can be moved out of a component and just into .ts code (maybe like `utils/config.ts`), returning an "isLoading", which App.tsx can use to either show the loader or show the Router.  Having a provider to do this (that's not really a React provider at all anymore) might confuse anyone new to the project.